storage/store/cache.go

```diff
@@ -0,0 +1,144 @@
+package store
+
+import (
+	"errors"
+	"fmt"
+
+	lru "github.com/hashicorp/golang-lru/v2"
+
+	"github.com/onflow/flow-go/module"
+	"github.com/onflow/flow-go/storage"
+)
+
+// nolint:unused
```

Comment:
do we need all of these `nolint` directives? seems like they will become out of date quickly.